virtual-setup/rstudio-login.md

```diff
@@ -67,6 +70,13 @@ You can stop your current R session with the orange circular on/off button in th
 <br>
 <br>
 
+RStudio may ask you if you would like to save your current workspace.
```

Comment:
We may need to add one more screenshot here, for when there are multiple files with changes. If they haven't saved their notebook yet, they may get a different dialog box, where we would want them to save the notebook, but not `.Rdata`